sources/packages/backend/apps/api/src/services/audit/audit.service.ts

```diff
@@ -0,0 +1,92 @@
+import { Injectable, LoggerService } from "@nestjs/common";
+import { InjectLogger } from "@sims/utilities/logger";
+import { AuditEvent } from "./audit-event.enum";
+import { AuthorizedParties } from "../../auth";
+
+const SIMS_AUDIT_EVENT_PREFIX = "SIMS Audit Event";
+
+@Injectable()
+export class AuditService {
+  /**
+   * Audit logging.
+   * @param clientIp client ip.
+   * @param tokenUserName token user name.
+   * @param event audit event.
+   * @param authorizedParty authorized party.
+   */
+  audit(
+    clientIp: string,
+    tokenUserName: string,
+    event: AuditEvent,
+    authorizedParty: AuthorizedParties,
+  ): void {
+    const eventFriendlyName = this.getEventFriendlyName(event);
+    const portalFriendlyName = this.getPortalFriendlyName(authorizedParty);
+    this.logger.log(
+      `${SIMS_AUDIT_EVENT_PREFIX} From ${clientIp} | User GUID: ${tokenUserName}, Event: ${eventFriendlyName}, Portal: ${portalFriendlyName}.`,
+    );
+  }
+
+  /**
+   * Gets an event friendly name to be logged given an audit event.
+   * @param event audit event.
+   * @returns audit event friendly name.
+   */
+  private getEventFriendlyName(event: AuditEvent): string {
+    let eventFriendlyName = "";
```

Comment:
Not a blocker, as a fallback it can be initialized with the `event`.
Same for the `getPortalFriendlyName`.